### PR TITLE
Update merge-rulesets.sh

### PR DESCRIPTION
**Pull request by koops76**

### CODE REVIEW

**Comment by koops76** [comment]
ping @Hainish @cowlicks.

File: utils/merge-rulesets.sh [hunk 1]
```diff
@@ -1,9 +1,11 @@
 #!/bin/sh
 
-# Merge all the .xml rulesets into a single "default.rulesets" file -- this
-# prevents inodes from wasting disk space, but more importantly, works around
-# the fact that zip does not perform well on a pile of small files.
+# Merge all the .xml rulesets into a single "default.rulesets" file -- this prevents inodes from wasting disk space, but more importantly,
+# works around the fact that zip does not perform well on a pile of small files.
```
Comment (Hainish) [inline utils/merge-rulesets.sh]: Why this change?

File: utils/merge-rulesets.sh [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
 
 python2.7 ./utils/merge-rulesets.py
 
 RULESETS=chrome/content/rules/default.rulesets
+
+# We make default.rulesets at build time, but it shouldn't have a variable timestamp.
+touch -r src/install.rdf src/$RULESETS
```
Comment (Hainish) [inline utils/merge-rulesets.sh]: I don't see how this is a substantive improvement.  Care to explain?

Comment (koops76) [reply utils/merge-rulesets.sh]: I moved this out of the Python script into the shell script.

File: utils/merge-rulesets.sh (hunk 1 above)
Comment (koops76) [reply utils/merge-rulesets.sh]: There's no need to limit line length to 80 chars.

Comment (koops76) [reply utils/merge-rulesets.sh]: All modern editors allow the user to choose the word wrap.

Comment (Hainish) [reply utils/merge-rulesets.sh]: Yet most styles still prefer it.  Take PEP 8, which instructs wrapping at 79 characters.  Wrapping a comment line can be quite ugly in some editors.  Let's keep the wrapping, it's orthogonal to the separate change in this PR.

Comment (koops76) [reply utils/merge-rulesets.sh]: @Hainish Done.

**Comment by Hainish** [comment]
Looks good, thanks @koops76
